inbm/integration-reloaded/test/ucc/UCC_GOOD_TELEMETRY_TEST.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -euxo pipefail # DO NOT REMOVE -- used to fail test if intermediate command fails
+
+source /scripts/test_util.sh
+
+start_time=$(get_time)
+print_all_error() {
+   echo "TEST FAILED!!!"
+}
+trap print_all_error ERR
+
+
+UCC_MSG='{"some": "arbitrary", "json": "string"}'
+
+test_echo Triggering Good UCC Telemetry Test
+listen_ucc_telemetry
+
+trigger_ucc "${UCC_MSG}"
+sleep 1
+if grep -q "arbitrary" ucc-telemetry-response;  then
```

Comment:
This is incorrect--it should check that the ucc telemetry response is equal to {"some": "arbitrary", "json": "string"}